lib/membrane/aac/parser/helper.ex

```diff
@@ -50,8 +50,9 @@ defmodule Membrane.AAC.Parser.Helper do
 
   defp parse_header(
          <<0xFFF::12, _version::1, 0::2, protection_absent::1, profile_id::2,
```

Comment:
Since you've touched that - I see 2 more differences with FFmpeg
1. The version bit is described as id
2. Next 2 is layer and doesn't have to be `0b00`, although FFmpeg always puts 0 there (https://github.com/FFmpeg/FFmpeg/blob/877ccaf776c92866e17493b48da49fab5b9ef11a/libavformat/adtsenc.c#L157)
See https://github.com/FFmpeg/FFmpeg/blob/877ccaf776c92866e17493b48da49fab5b9ef11a/libavcodec/adts_header.c#L41

Reply:
Indeed, I believe the specification says to set the layer bits to `0b00`, but I have changed it following your advice